docs/src/main/asciidoc/qute-reference.adoc

```diff
@@ -0,0 +1,698 @@
+<2> Loop with an explicit alias `item`.
+
+[TIP]
+====
+Data passed to the template instance are always accessible using the `data` namespace. 
+This could be useful to access data for which the key is overriden:  
+
+[source,html]
+----
+<html>
+{item.name} <1>
+<ul>
+{#for item in item.getDerivedItems()} <2>
+  <li>
+  {item.name} <3> 
+  is derived from
+  {data:item.name} <4>
+  </li>
+{/for}
+</ul>
+</html>
+----
+<1> `item` is passed to the template instance as a data object.
+<2> Iterate over the list of derived items.
+<3> `item` is an alias for the iterated element.
+<4> Use the `data` namespace to access the `item` data object.
+
+====
+
+==== Sections
+
+A section:
+
+* has a start tag
+** starts with `#`, followed by the name of the section such as `{#if}` and `{#each}`,
+* may be empty 
+** tag ends with `/`, ie. `{#emptySection /}`
+* may contain other expression, sections, etc.
+** the end tag starts with `/` and contains the name of the section (optional): `{#if foo}Foo!{/if}` or `{#if foo}Foo!{/}`,
+
+The start tag can also define parameters.
+The parameters have optional names.
+A section may contain several content *blocks*. 
+The "main" block is always present.
+Additional/nested blocks also start with `#` and can have parameters too - `{#else if item.isActive}`.
+A section helper that defines the logic of a section can "execute" any of the blocks and evaluate the parameters.
+
+[source]
+----
+{#if item.name is 'sword'}
+  It's a sword!
+{#else if item.name is 'shield'}
+  It's a shield!
+{#else}
+  Item is neither a sword nor a shield.
+{/if}
+----
+
+===== Loop Section
+
+The loop section makes it possible to iterate over an instance of `Iterable`, `Map` 's entry set and `Stream`.
+It has two flavors.
+The first one is using the `each` name alias.
+
+[source]
+----
+{#each items}
+ {it.name} <1>
+{/each}
+----
+<1> `it` is an implicit alias. `name` is resolved against the current iteration element.
+
+The other form is using the `for` name alias and can specify the alias used to reference the iteration element:
+
+[source]
+----
+{#for item in items}
+  {item.name}
+{/for}
+----
+
+It's also possible to access the iteration metadata inside the loop:
+
+[source]
+----
+{#each items}
+ {count}. {it.name} <1>
+{/each}
+----
+<1> `count` represents one-based index. Metadata also include zero-based `index`, `hasNext`, `odd`, `even`.
+
+===== If Section
+
+A basic control flow section.
+The simplest possible version accepts a single parameter and renders the content if it's evaluated to `true` (or `Boolean.TRUE`).
+
+[source]
+----
+{#if item.active}
+ This item is active.
+{/if}
+----
+
+You can also use the following operators:
+
+|===
+|Operator |Aliases 
+
+|equals 
+|`eq`, `==`, `is` 
+
+|not equals 
+|`ne`, `!=`
+
+|greater than 
+|`gt`, `>`
+
+|greater than or equal to
+|`ge`, `>=`
+
+|less than 
+|`lt`, `<`
+
+|less than or equal to
+|`le`, `<=`
```

Comment:
I just checked the generated output and this gets replaced by an arrow by Asciidoctor. It needs some escaping.

Using `\<=` does the trick.

Reply:
Oh, I didn't know about this one. Thanks!